src/guides/en/understand-devtools/index.md

```diff
@@ -0,0 +1,236 @@
+* Tool: a feature of DevTools, specialized in a particular web development task, which appears inside a panel.
+* Sidebar: a vertical bar on the left or right side of a tool's content, within a panel.
+* Drawer: a horizontal, toggleable, area, which appears at the bottom and optionally contains a second toolbar for accessing other tools.
+
+### High-level UI description
+
+DevTools appears next to the rendered webpage, in a browser tab. It might appear to the right of the webpage:
+
+![A browser window, with a rendered webpage on the left, and DevTools on the right](./browser-devtools-right.png)
+
+Or it might appear at the bottom of the webpage:
+
+![A browser window, with a rendered webpage on the top, and DevTools at the bottom](./browser-devtools-bottom.png)
+
+Now let's focus on just DevTools. The following screenshot shows DevTools only, without the rendered webpage next to it or the browser UI around them. The screenshot shows the toolbar at the top of DevTools, with tabs, one tab per tool. The bottom part contains a panel that displays the content of the currently selected tool:
+
+![DevTools with the toolbar at the top, and a panel at the bottom](./devtools-tabs.png)
+
+It's also possible for two tools to be visible at once, with one in the panel, and one in a toggleable drawer at the bottom. Often, the tool displayed in the drawer is the **Console** tool, which is used to log messages and run JavaScript code.
+
+![DevTools with the toolbar at the top, a panel in the middle, and a drawer at the bottom](./devtools-drawer.png)
+
+Let's zoom in one more level and focus on the content of the tool that's displayed in the panel. The following screenshot ignores the toolbar and drawer, and only shows the content of the panel where a tool is displayed. Each tool is different, and they don't all look the same, but the following screenshot shows the main elements that are often present in a tool: a sidebar on the left, with tabs used to display different sections of the tool, and a main content area on the right, which displays the content of the currently selected section:
+
+![A single DevTools tool, with a sidebar on the left, and a main content area on the right](./devtools-tool-with-sidear.png)
+
+Now let's put it all back together in one screenshot, so you can see how everything fits together. The screenshot shows the entire browser window, with a rendered webpage on the left, and DevTools on the right. DevTools has a toolbar, with tabs, a panel showing the selected tool, and a drawer area at the bottom. The panel shows a tool which contains a sidebar, and a main content area:
+
+![The browser with DevTools](./devtools-all-in-one.png)
+
+This might look like a lot of UI elements, and hard to understand at first. But, hopefully, with the way we've broken it down step by step, it helps making sense of the UI and forming a mental model of how it's organized.
+
+### Differences between browsers
```

Comment:
you already make the same point below "The UI of DevTools" so this is a little redundant.